shark/examples/shark_inference/stable_diffusion/stable_args.py

```diff
@@ -123,6 +123,12 @@ def path_expand(s):
     help="other supported schedulers are [PNDM, DDIM, LMSDiscrete, EulerDiscrete, DPMSolverMultistep]",
 )
 
+p.add_argument(
+    "--output_dir",
+    type=str,
+    default=None,
```

Comment:
should this be `pwd`

Reply:
yes, it is pwd or the path specified by the user.
taken into consideration in main.py
also it will create a path specified by user if not already created